packages/@okta/vuepress-site/docs/guides/pwd-optional-widget-sign-in-email/main/nodeexpress/integrationsummary.md

```diff
@@ -1,5 +1,9 @@
 <div class="full">
 
-![Sequence diagram illustrating the Sign-In Widget's password-optional sign-in use case.](/img/pwd-optional/pwd-optional-widget-sign-in-summary.png)
+![Sequence diagram illustrating the Sign-In Widget's password-optional sign-in use case.](/img/pwd-optional/pwd-optional-node-widget-sign-in-summary.png)
```

Comment:
Change "Display verify with an email page" so it sounds less awkward.

Reply:
Same comment as above.
"Display with your email"?